examples/with-drizzle-pagesdir/src/components/uploader.tsx

```diff
@@ -0,0 +1,32 @@
+import { UploadButton, UploadDropzone } from "~/utils/uploadthing";
+
+export function Uploader() {
```

Comment:
this doesn't have to be a separate file in this example since we don't have to separate client components